src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml

```diff
@@ -14,7 +14,7 @@
         @Html.HiddenFor(m => m.Type)
         @Html.HiddenFor(m => m.Status, new { id = "status-id" })
         @Html.HiddenFor(m => m.Reference)
-        @Html.HiddenFor(m => m.Reason)
+        @*@Html.HiddenFor(m => m.Reason)*@
```

Comment:
This would have been added for a reason. I suggest we add it back in but update for m.RejectedReason and m.ReturnedReason, I suspect the value will vanish if validation error happens.